trino/client.py

```diff
@@ -606,11 +665,11 @@ def raise_response_error(self, http_response):
         raise exceptions.HttpError(
             "error {}{}".format(
                 http_response.status_code,
-                ": {}".format(http_response.content) if http_response.content else "",
+                ": {!r}".format(http_response.content) if http_response.content else "",
```

Comment:
what does this do?

Reply:
`!r` calls repr() on the expression ([source](https://peps.python.org/pep-0498/#specification))
PEP498 mention, that this is redundant ([source](https://peps.python.org/pep-0498/#s-r-and-a-are-redundant))

but without it, mypy throws error:
```
error: On Python 3 formatting "b'abc'" with "{}" produces "b'abc'", not "abc"; use "{!r}" if this is desired behavior  [str-bytes-safe]
```

Reply:
Does mypy stop complaining if you add repr around the argument to format instead of changing format string? I think it'll be more obvious.

Reply:
It works, changed according to you suggestion